pages/index.tsx

```diff
@@ -10,32 +11,50 @@ import { AUTH_COOKIE_NAME } from 'shared/constants';
 
 interface Props {
   isAuthed: boolean;
+  seenOnBoarding: boolean;
 }
 
 const Index: NextPage<Props> = props => {
   const [isAuthed, setIsAuthed] = React.useState(props.isAuthed);
+  const [seenOnBoarding, setSeenOnBoarding] = React.useState(props.seenOnBoarding);
+  const router = useRouter();
+
+  // to avoid this error:
+  // During Pre-rendering (SSR or SSG) you tried to access a router method push, replace, back, which is not supported.
+  React.useEffect(() => {
+    const { query } = router;
+    if (query.seenOnBoarding === 'true') {
+      setSeenOnBoarding(true);
+      // reset query parameters once user gets back to home page
+      router.replace({
+        pathname: '/',
+        query: {},
+      });
+    } else if (!seenOnBoarding && isAuthed) {
+      router.push('/onBoarding');
+    }
+  }, []);
 
   const onLogout = async () => {
     const axios = createAxiosInstance();
     await axios.post('/api/auth/logout');
     setIsAuthed(false);
+    setSeenOnBoarding(false);
   };
 
   return (
-    <Container maxWidth="lg" className="flex justify-between pt-4">
-      <Typography variant="h1" className="text-3xl">
+    <Container maxWidth='lg' className='flex justify-between pt-4'>
```

Comment:
why the quote change? JSX should use double quotes to imitate HTML conventions

Reply:
Oh, didn't even consider adding some JSX rules to ESLint... 🤦🏼‍♂️😅

Reply:
was this your linting change 🥲

Reply:
No, it wasn't! I actually panicked when I saw his changes, went and double-checked! 😂

Reply:
My bad, prettier was set to single quotes for jsx, didn't know it was a convention to use double quotes with JSX